test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+set -a
+
+: ${RUBY_VERSION=2.2}
+sed "s/\${RUBY_VERSION}/$RUBY_VERSION/" Dockerfile > Dockerfile.$RUBY_VERSION
```

Comment:
oh `sed`, you faithful steed.

Will be great when we can avoid this w/ new Docker version, as you commented above.